apps/app-article-server/README.md

```diff
@@ -19,7 +19,7 @@ yarn start
 
 As an alternative to exporting PUBLIC_URL for every new shell you can also use [direnv](https://direnv.net/docs/installation.html) to have it done automgically.
 
-A server in port `8080` should be now up. Here's an example article for you to test it `http://localhost:8080/article/44f345b5-eb92-4e60-a884-26396cd3d831`.
+A server in port `8003` should be now up. Here's an example article for you to test it `http://localhost:8003/article/44f345b5-eb92-4e60-a884-26396cd3d831`.
```

Comment:
The default seems to be 8080 (see src/server/index.js). I guess that default should be changed.

Reply:
In `.env` it is `8003`.